portfolio/src/main/webapp/style.css

```diff
@@ -1,3 +1,35 @@
+body {
+  /* background-image: url("/images/background.jpg");
+  background-size: cover; */
+  background-color: darkblue;
+  color: aliceblue;
+}
+
+header li {
+  display:inline
+}
+
+header h1 {
+  font-size: xxx-large;
+}
+
+header img {
+  height: 1rem;
+  width: 1rem;
+  border-radius: 0.5rem;
+}
```

Comment:
I  recommend putting the css rules in alphabetical order. That way it is easy to scan a rule quickly to see what the properties are. This is helpful when you have a lot of properties in  a rule. I would do that throughout the file.

header img {
  border-radius:
  height:
  width:
}

Reply:
done